### DISCUSSION TITLE
Localization

### DISCUSSION THREAD

**Discussion by Trinitou** [free-audio/clap#224] [Ideas]
What about localization features?

It might be interesting if the host could request the plugin to show all strings in a certain language (using two-letter or three-letter country codes). Also parameter names could adapt. And what about unicode strings?

Just want to get a discussion started ;)

**Comment by baconpaul** [comment]
So all strings in CLAP are utf-8 so unicode is already there. You can name your parameter "Spicy 🥓 Cutoff" (as long as you use the UTF-8 encoding) and it will work with CLAP. 

Localization: doesn't the `std::locale` tell you your localization? Do we need that in the CLAP API? I think if a plugin supports localization the right thing to do is (1) call std::locale; (2) see if you have support for that language and fall back if not (presumably to `en_US` which seems the defacto standard in plugin land, but i suppose to whatever you want your base language to be) and (3) if you do call `params::paramInfoChanged` with `INFO` and the new UTF-8 strings.
